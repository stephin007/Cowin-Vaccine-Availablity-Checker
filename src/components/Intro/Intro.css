.intro__home > img {
  width: 550px;
}

.intro__home {
  display: flex;
  align-items: center;
  flex-direction: column;
  justify-content: center;
  margin-right: 40px;
}

.intro__homeText {
  border: 3px solid black;
  padding: 10px 55px 10px 50px;
  margin-top: -25px;
  font-weight: 400;
  text-align: center;
}

.intro__homeText:hover {
  border: 3px solid black;
  color: white;
  padding: 10px 55px 10px 50px;
  margin-top: -25px;
  transition: 0.5s;
  background-color: black;
  font-weight: 400;
  text-align: center;
}

@media screen and (max-width: 800px) {
  .intro__home > img {
    width: 300px;
  }
  .intro__home {
    display: flex;
    align-items: center;
    flex-direction: column;
    justify-content: center;
    margin-right: -10px;
    margin-top: 40px;
  }
  .intro__home {
    display: flex;
    align-items: center;
    flex-direction: column;
    justify-content: center;
    margin-left: 20px;
  }
}
