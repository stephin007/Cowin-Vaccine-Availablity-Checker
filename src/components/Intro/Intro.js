import React from "react";
import { Link } from "react-router-dom";
import "./Intro.css";

const Intro = () => {
  return (
    <div className="intro">
      <div className="intro__home">
        <img
          src="https://user-images.githubusercontent.com/71087810/117496553-d5af7b00-af94-11eb-84bb-913a1f386811.png"
          alt=""
        />

        <Link
          style={{ textDecorationLine: "none", color: "black" }}
          to="/vaccines"
        >
          <h3 className="intro__homeText">Vaccine Availability</h3>
        </Link>
      </div>
    </div>
  );
};

export default Intro;
