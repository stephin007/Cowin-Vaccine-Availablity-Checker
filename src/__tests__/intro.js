import { shallow } from 'enzyme';
import React from 'react';
import Intro from '../components/Intro/Intro';

describe('renders intro component', () => {
	const wrapper = shallow(<Intro />);

	it('renders intro correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});

	it('renders home text', () => {
		expect(wrapper.find('.intro__homeText').text()).toEqual('Vaccine Availability');
	});
});
