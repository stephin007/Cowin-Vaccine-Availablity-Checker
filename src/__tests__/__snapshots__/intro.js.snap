// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders intro component renders intro correctly 1`] = `ShallowWrapper {}`;
